core/src/main/scala/akka/kafka/scaladsl/RelativeOffsetAssignmentHandler.scala

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2014 - 2016 Softwaremill <http://softwaremill.com>
+ * Copyright (C) 2016 - 2019 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.kafka.scaladsl
+
+import akka.kafka.RestrictedConsumer
+import org.apache.kafka.common.TopicPartition
+
+import scala.jdk.CollectionConverters._
+
+object RelativeOffsetAssignmentHandler {
+  def apply(backOffset: Long): RelativeOffsetAssignmentHandler =
+    new RelativeOffsetAssignmentHandler(None, Some(backOffset))
+
+  def apply(topicPartition: TopicPartition, backOffset: Long): RelativeOffsetAssignmentHandler =
```

Comment:
It seeks given partition to offset = end offset - backOffset and all others are assigned to beginning offset.